docs/ldml/tr35-general.md

```diff
@@ -818,29 +818,48 @@ This data is supplied in **Part 6: [Supplemental](tr35-info.md#Contents)**: [Sec
+Units of measurement, such as _meter_, have defined programmatic identifiers as described in this section. 
+The main identifier is a _core unit identifier_, which encompasses a number of simpler types of identifiers as follows. 
+A secondary type of identifier is a _mixed unit identifier_, which combines a series of units such as _5° 30′_ or _3 feet 7 inches_.
 
 | Name             | Examples |
 | ---------------- | -------- |
-| simple unit ID   | meter, foot, inch, pound, pound-force, … |
-| prefixed unit ID | kilometer, centigram, … <br/> _plus simple unit IDs_ |
-| single unit ID   | square-foot, cubic-centimeter, … <br/> _plus prefixed unit IDs_ |
 | core unit ID     | kilometer-per-hour, kilogram-meter, kilogram-meter-per-square-second, … <br/> _plus single unit IDs_ |
-| mixed unit ID    | foot-and-pound |
+| single unit ID   | square-foot, cubic-centimeter, … <br/> _plus prefixed unit IDs_ |
+| prefixed unit ID | kilometer, centigram, … <br/> _plus simple unit IDs_ |
+| simple unit ID   | meter, foot, inch, pound, pound-force, … |
+| mixed unit ID    | foot-and-inch, degree-and-arc-minute-and-arc-second |
+
+
+There is currently a ‘long’ style of unit identifier corresponding to each _core unit identifier_, as illustrated below. 
+The only difference is that the long unit identifier adds a prefix which was used in the CLDR Survey Tool for grouping related identifiers together. 
+The long unit identifers are used as a key in the translated unit names for locales, but dealing with these two styles is unnecessarily complicated, so the long unit identifiers are slated for deprecation (after replacing their use as a key for translations).
+
+| core unit ID | long unit ID |
+| ------------ | ------------ |
+| meter        | length-meter |
+| pound        | mass-pound   |
+| day          | duration-day |
  
-A core unit that is not a simple unit is called a _complex unit_. It is valid to construct a complex unit identifier from multiple simple unit identifiers using multiplication (kilogram-meter) and division (kilogram-per-meter). As usual, with division the part before the (first) -per- is called the _numerator_, and the part after it is called the _denominator_.
 
-The conversion information uses the short unit identifiers, discarding the unitType. Thus “meter” is used instead of “length-meter”. The translation data currently uses the long unit identifiers, for backwards compatibility. However, that is likely to change in a future version.
+The list of valid CLDR simple unit identifiers is found in _Section [3.11 Validity Data](tr35.md#Validity_Data)_. 
+These names should not be presented to end users, however: the translated names for different languages (or variants of English) are available in the CLDR localized data. 
```

Comment:
> These names should not be presented to end users, however: the translated names for different languages (or variants of English) are available in the CLDR localized data. 

OK, I see this was moved rather than dropped. That's good, although I think this information might want to be right up at the top where these identifiers are introduced.